strapi/strapi
# chore: remove js-cookie dependency

### What does it do?

It removes the js-cookie dependency

### Why is it needed?

js-cookie is an unnecessary dependency for our use case.

### How to test it?

Run `yarn test:front packages/core/admin/ee/admin/tests/cookies.test.ts ` in the monorepo directory to check that the cookie helper functions do what is needed.

To check it in a Strapi project, make sure it is EE, and you have configured and SSO Provider. Do the login and and the logout and check it works. Check also that after doing the login the localStorage has the cookie value and the cookie value is deleted.

### Related issue(s)/PR(s)

https://strapi-inc.atlassian.net/browse/DX-1250
